docs/building-apps/network-features/vetkeys/api.mdx

```diff
@@ -0,0 +1,147 @@
+import TabItem from "@theme/TabItem";
+import { AdornedTabs } from "/src/components/Tabs/AdornedTabs";
+
+# vetKD API
+
+As described in the [previous section](/docs/building-apps/network-features/vetkeys/introduction), vetKD is the protocol for deriving vetKeys.
+The management canister provides the following low-level APIs for the vetKD protocol.
+
+```candid
+  vetkd_public_key : (record {
+    canister_id : opt canister_id;
+    context : blob;
+    key_id : record { curve : vetkd_curve; name : text };
+  }) -> (record { public_key : blob; });
+  vetkd_derive_key : (record {
+    input : blob;
+    context : blob;
+    transport_public_key : blob;
+    key_id : record { curve : vetkd_curve; name : text };
+  }) -> (record { encrypted_key : blob; });
+```
+
+The following key names are supported:
+
+- `dfx_test_key`: Only available on the local development environment started by dfx.
+
+- `test_key_1`: Test key available on the ICP mainnet.
+
+- `key_1`:  Production key that will soon be available on the ICP mainnet.
```

Comment:
In addition to the list of keys, I would expand with some explanation of the various parameters in the two APIs, e.g.:

* Input: arbitrary data used to derive keys. We can probably say that it can be used to identify the key in the canister as distinct inputs will result in distinct derived keys, we could also mention that it does not need to be random.  
* Context: used as a domain separator. This ensures the key is derived within a specific context, avoiding collisions across different applications
* transport_public_key: The recipient’s public key, used to encrypt the derived key for secure transport.
* key_id: Identifies the master key from which the derived key is created.